### ISSUE TITLE
cannot change default language

### ISSUE THREAD

**Issue by calimou** [datenstrom/yellow#122]
I've created the file `language-fr.ini` (with the provided fr yellow language file) and updated `config.ini` to `Language: fr`, but I still get the english text labels. What could I miss ?

Followed the instructions found at https://github.com/datenstrom/yellow-extensions/tree/master/languages/fran%C3%A7ais

**Comment by nibreh** [comment]
Salut. Tu as raison, je viens de tester. Tu peux confirmer que le panneau de connexion est en français, mais après ça, le menu est toujours en anglais ?

Ok now I write in english :)

I tested with a new installation and yes, languages files are not working anymore. 
The connexion panel is OK, but after connexion Yellow menu is still in English (Logout is in english, etc)
I made some changes in language-en.ini for testing, Yellow is still looking at that file ?
